templates/repo/view_file.tmpl

```diff
@@ -1,4 +1,4 @@
-<div class="{{TabSizeClass .Editorconfig .FileName}} non-diff-file-content">
+<div {{if .ReadmeInList}}id="readme"{{end}} class="{{TabSizeClass .Editorconfig .FileName}} non-diff-file-content">
```

Comment:
The tiniest of nitpicks, this adds a double space if the readme is not present. I'm on mobile right now, but I think the approach that is taken in other files is having one of those spaces be inside the conditional

Reply:
Fixed.